definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+ Definition of done
+
+This document clarifies, when an product backlog item is ready. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api (diagrams?)
+* [ ] Tutorials
+* [ ] Readme
+* [ ] ROS wiki
+* [ ] Code (sphinx/doxygen)
+* [ ] Good commit messages ([example](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9
+))
+* [ ] Changelog.md updated
+* [ ] Copyright headers
+
+## Reviews
+* [ ] Architecture
+* [ ] Code (coding rules, style guide)
+* [ ] Test review (test plan and individual test cases)
+* [ ] Documentation review
+
+## Quality: CI pass
+* [ ] Catkin_lint
+* [ ] Coverage
+* [ ] Tests
+
+## Demo
+* [ ] Example program / snippet prepared
+* [ ] Ready for sprint review
+
+## Release planning
+* [ ] When is the new feature released?
+
+## Cleanup
+* [ ] Merge
```

Comment:
Maybe add a comment to squash unnecessary commits to avoid a cluttered git history?
For example see the [GitHub blog](https://github.blog/2016-04-01-squash-your-commits/).

Reply:
Done.